File: go/day25.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
	"strings"
)

func add(x int, y int) int {
	return x + y
}

func main() {
	fmt.Print("Input 2 numbers: ")

	reader := bufio.NewReader(os.Stdin)
	input, _ := reader.ReadString('\n')
	input = strings.TrimSpace(input)

	num1, _ := strconv.Atoi(input);

	
	reader2 := bufio.NewReader(os.Stdin)
	input2, _ := reader2.ReadString('\n')
	input2 = strings.TrimSpace(input2)

	num2, _ := strconv.Atoi(input2)

	fmt.Print(
		"Sum: " + 
		strconv.Itoa(add(num1, num2)),	
	);

}

File: cSharp/day25.cs

using System;

public static int add(
    int x,
    int y
) {
    return x + y;
}

Console.WriteLine("Input 2 numbers");


string input1 = Console.ReadLine();
string input2 = Console.ReadLine();

int.TryParse(input1, out int numInput1);
int.TryParse(input2, out int numInput2);

Console.WriteLine(
    "Sum: " +
    add(numInput1, numInput2).ToString()
);